webpack.config.js

```diff
@@ -36,6 +36,22 @@ module.exports = function(env = {}) {
 
   const nodeModsDir = path.resolve(__dirname, 'node_modules');
   const outputPath = path.resolve(__dirname, 'dist');
+
+  if (env.dev) {
+    const vendorManifest = path.join(
+      __dirname,
+      './dist',
+      'vendor-manifest.json'
+    );
+
+    plugins.push(
+      new webpack.DllReferencePlugin({
+        manifest: require(vendorManifest),
```

Comment:
This require could potentially fail, maybe `require` above in `try/catch`?

Reply:
good call